### ISSUE TITLE
Pinch does not work on child element

### ISSUE THREAD

**Issue by zhuochen83** [react-spring/react-use-gesture#84]
Hi I am trying to make use pinch to zoom on an element with some child element. 

Here is an example I made with CRA. If you have one finger inside the red box one finger outside the box, pinch does not work.

```
import React from 'react';
import { useGesture } from "react-use-gesture";
import { useSpring, animated } from "react-spring";
import logo from './logo.svg';
import './App.css';

const App: React.FC = () => {
  const [{ scale }, update] = useSpring(() => ({
    scale: 1,
}));

const getScale = (momentum: number) => Math.max(0.1, ((scale.getValue() as number) + momentum));

  const binding = useGesture({
    onPinch: ({ vdva }) => {
        update({ scale: getScale(vdva[0]) });
    },
});

  return (
    <div {...binding()} className="App">
      <animated.div style={{
        transform: scale.interpolate(s => `scale(${s})`),
      }}>
        <header className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <div  style={{width: 150, height: 150, backgroundColor: 'red'}}></div>        
        </header>
      </animated.div>
    </div>
  );
}

export default App;
```

**Comment by dbismut** [comment]
Hi @zhuochen83 I'm not sure about your implementation to be honest.
Can you try this and let me know if this is what you want:
https://codesandbox.io/s/react-use-gesture-template-hwyjz

If not, can you please create a sandbox?

**Comment by zhuochen83** [comment]
Thanks @dbismut for the quick reply
Here is my sandbox.
https://codesandbox.io/s/react-use-gesture-template-4b0wv
I am using react-use-gesture: 5.2.4 while your example uses 6.0.0-beta-3. This may be why mine doesn't work

**Comment by dbismut** [comment]
Hm 6.0.0 doesn’t have much internal differences for now, it just does some renaming of state props.

What I don’t understand is how and why you handle scale with velocity?

**Comment by zhuochen83** [comment]
Yes, delta works perfectly. I will use that instead.

Correct me if I am wrong. I would still think velocity should work in this case. For some reason, the event got triggered but the velocity would not change.

**Comment by dbismut** [comment]
> Correct me if I am wrong. I would still think velocity should work in this case. For some reason, the event got triggered but the velocity would not change.

@zhuochen83 I've just published `6.0.0-beta.4` that should prevent `usePinch / onPinch` to run twice on `touchmove` for the same event. This would make velocity equal to `0` every two frames which is what you're experiencing I think. However, I can't make sense of your formula using velocity, you're converting pixels per millisecond to pixels so that can't work properly...

Note that beta version is currently pretty stable and corrects some bugs ;) Closing this but feel free to reopen if you're unsatisfied with this answer.
